cmd/cnbBuild.go

```diff
@@ -269,6 +279,7 @@ func runCnbBuild(config *cnbBuildOptions, telemetryData *telemetry.CustomData, u
 		if (config.Buildpacks == nil || len(config.Buildpacks) == 0) && len(descriptor.Buildpacks) > 0 {
 			config.Buildpacks = descriptor.Buildpacks
 		}
```

Comment:
There are multiple ways to provide Buildpacks. We should report all of those and in all cases. Reporting the buildpacks actually used - in addition to all buildpacks related config - probably makes sense anyway.